### ISSUE TITLE
Player decides to move out of the map

### ISSUE THREAD

**Issue by polltery** [polltery/node-cli-game-example#13]
Right now when a player is at the edge of the map (e.g. position `0,0`) they are **not** supposed to move out of the map. For example below, the player is not given the option to move north, however they can still decide to move north, and if they do.. the game crashes.
```
TURN : 1
MAP VIEW
 *  *  *  * 
 *  *  *  * 
 *  *  *  * 
 *  *  *  * 
you are at 0,0
Here are your movement options..
S(South) : unknown
E(East) : unknown
SE(South-East) : unknown
```

We can fix this either by placing a check after player input or by using [RegExp](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp) objects for prompt `pattern`.
```
    prompt.get({
        description : 'Please enter a movement option',
        type : 'string',
        pattern : customRegex,
        message : 'Please choose an appropriate movement option',
        require : true,
        before : val => val.toUpperCase()
```

**Comment by sbab** [comment]
I can work on this if you like :)

**Comment by sbab** [comment]
Should be fixed by #17. Happy Hacktoberfest!
